crates/sozo/ops/src/migration/migrate.rs

```diff
@@ -882,6 +874,89 @@ pub async fn update_manifests_and_abis(
     Ok(())
 }
 
+pub async fn find_authorization_diff<A>(
```

Comment:
If you could add a comment that for now we focus on writers, and owners will be added later during a refactor that could be great to not forget it. 👍